README.md

```diff
@@ -1 +1,68 @@
-# go-lib-micro
\ No newline at end of file
+# go-lib-micro
+[![Build Status](https://travis-ci.org/mendersoftware/go-lib-micro.svg?branch=master)](https://travis-ci.org/mendersoftware/go-lib-micro)
+[![codecov](https://codecov.io/gh/mendersoftware/go-lib-micro/branch/master/graph/badge.svg)](https://codecov.io/gh/mendersoftware/go-lib-micro)
+[![Go Report Card](https://goreportcard.com/badge/github.com/mendersoftware/go-lib-micro)](https://goreportcard.com/report/github.com/mendersoftware/go-lib-micro)
+
```

Comment:
ok, I dodn't 'forget', we just didn't do it until now.

it doesn't look helpful:
https://godoc.org/github.com/mendersoftware/go-lib-micro

it's probably because no code was commited yet.

I added it and we can see what happens on first merge.